### ISSUE TITLE
question: runner wrapper functions missing (non kw) argument handling

### ISSUE THREAD

**Issue by gladiatr72** [saltstack/salt#36760]
### Description of Issue/Question

`modules.saltutil.runner`, `states.saltmod.runner` cannot be used to execute runner functions that have non-keyword arguments.  Is there a particular reason it is thus?  

`salt.runner.Runner.cmd()` is the ultimate workhorse for these calls

```
In [17]  rclient.cmd?
Signature: rclient.cmd(fun, arg=None, pub_data=None, kwarg=None, print_event=True)
Docstring: Execute a function
File:      /usr/lib/python2.7/dist-packages/salt/runner.py
Type:      instancemethod
```

`saltutil.runner()` `rclient.cmd` call:
```
def runner(name, **kwargs):
    [...]
    return rclient.cmd(name, kwarg=kwargs, print_event=False)
```

### Versions Report
Salt Version:
           Salt: 2016.3.3

Dependency Versions:
           cffi: 1.7.0
       cherrypy: 3.2.2
       dateutil: 2.5.3
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.21.1
           Mako: 0.9.1
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.2.3
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
   python-gnupg: Not Installed
         PyYAML: 3.10
          PyZMQ: 15.4.0
           RAET: Not Installed
          smmap: 0.8.2
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.5

System Versions:
           dist: Ubuntu 14.04 trusty
        machine: x86_64
        release: 3.13.0-95-generic
         system: Linux
        version: Ubuntu 14.04 trusty

**Comment by gtmanfred** [comment]
I believe that the answer is that it is better to be explicit, but I am not sure.

But let me check around.

**Comment by gtmanfred** [comment]
Yup, there doesn't appear to be a specific reason to only be using keyword arguments, other than that is the way it was originally written.
